db/migrate/20200412211316_create_garden_plants.rb

```diff
@@ -0,0 +1,8 @@
+class CreateGardenPlants < ActiveRecord::Migration[5.2]
+  def change
+    create_table :garden_plants do |t|
```

Comment:
Don't we need a date stamp or something here? Or is the plan to do this in a separate migration later?

Reply:
We could.  are you thinking for the calendar generation?

Reply:
yes, but maybe a separate migration when we are ready